jsonnet/components/alertmanager.libsonnet

```diff
@@ -319,6 +332,56 @@ function(params)
                 mountPath: '/etc/tls/private',
                 name: 'secret-alertmanager-main-tls',
               },
+              {
+                mountPath: '/etc/tls/client',
+                name: 'metrics-client-ca',
+                readOnly: false,
+              },
+            ],
+          },
+          {
+            name: 'kube-rbac-proxy-metric',
```

Comment:
Why do we need another kube-rbac-proxy container?

Reply:
I tried to mix 2 authorizer in the same config for kube-rbac-proxy but it is not working well.
I raised an issue for kube-rbac-proxy: https://github.com/brancz/kube-rbac-proxy/issues/146
Waiting for a solution, I put a dedicated proxy for the metric endpoints.

Reply:
Of course, just added a TODO mentioning this issue and explaining why this container is added.